Assets/MixedRealityToolkit.Extensions/Sharing/Localization/SpatialLocalizer.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.Sharing
+{
+    /// <summary>
+    /// Very simple consumer of <see cref="ISpatialCoordinate"/> to demonstrate usage.
+    /// </summary>
+    public class SpatialLocalizer : MonoBehaviour
```

Comment:
>SpatialLocalizer [](start = 17, length = 16)

nit: may be more appropriate to call this out as a coordinate localizer. There may one day be more elaborate helper classes that uses multiple coordinates for positioning items.

Reply:
Good call

---
In reply to: [270513793](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3732#discussion_r270513793) [](ancestors = 270513793)